src/main/resources/db/migration/V1_5__add_expired_columns_to_varsel_table.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE varsel
+    ADD COLUMN expires_at TIMESTAMPTZ DEFAULT NULL;
```

Comment:
Burde denne være `NOT NULL` ? Eller tenker vi det kan komme varsler som ikke har frist?

Reply:
Den synes jeg bør vært `NOT NULL` ja! 🙌